addons/core/functions/fnc_getDefaultRadioSettings.sqf

```diff
@@ -26,23 +26,20 @@
   Public: Yes
 */
 
-params [["_isSW", true, [false]], ["_unit", TFAR_currentUnit, [objNull]]];
+params [
+    ["_isSW", true, [false]],
```

Comment:
Change to SR. As that should be the new terminology.. Atleast I'd prefer that. I'll leave choice to you